### DISCUSSION TITLE
How to enable Auto dark mode according to OS setting on this theme?

### DISCUSSION THREAD

**Discussion by mendax1234** [squidfunk/mkdocs-material#5745] [Q&A] [answered]
How to enable Auto dark mode according to OS setting on this theme?

**Comment by lampensau** [comment, marked as answer]
You'll need a version of mkdocs-material-insiders for the built-in solution, as noted in the [documentation](https://squidfunk.github.io/mkdocs-material/setup/changing-the-colors/#automatic-light-dark-mode). The feature will generally be available when reaching the next funding goal [`Goat's Horn`](https://squidfunk.github.io/mkdocs-material/insiders/#14000-goats-horn).

Additional references: 

- Issue: #3244 
- Third party plugin: [mkdocs-system-preference-theme-toggle](https://github.com/jannismain/mkdocs-system-preference-theme-toggle)

**Reply by mendax1234** [reply]
Thanks! l notice that the blog plugin has been opened to the public, how can l use it? After reading the official guide, l still don't find how to install the `blog` plugin. Should l use `pip install mkdocs-blog-plugin`?

**Reply by lampensau** [reply]
The blog plugin will be available with the release of mkdocs-material 9.2, which is currently in beta.

Please check out the issue #5686 for more info on the beta.

**Reply by mendax1234** [reply]
Thanks so much! @lampensau Do you know when the mkdocs-material 9.2 will be officially released? Besides, which Python version is recommended to use mkdocs-material?

**Reply by squidfunk** [reply]
We'll release it as a stable version in the coming weeks, but we don't have a date set yet. It's a very big and important release, so please give us some time. However, the beta should be perfectly fine to use. If you fine problems, please report them in #5686.
